Uno/NuGetPackageExplorer.Shared/Legacy/Presentation/ShellPageViewModel.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Input;
+
+using Microsoft.Extensions.DependencyInjection;
+using Uno.Foundation;
+using NupkgExplorer.Framework.MVVM;
+using NupkgExplorer.Framework.Navigation;
+using NupkgExplorer.Presentation.Content;
+
+namespace NupkgExplorer.Presentation
+{
+    public class ShellPageViewModel : ViewModelBase
+    {
+        public ViewModelBase ActiveContent { get => GetProperty<ViewModelBase>(); set => SetProperty(value); }
+
+        public bool IsStandaloneWindow
+        {
+            get => GetProperty<bool>();
+            set => SetProperty(value);
+        }
+
+        public ShellPageViewModel()
+        {
+
+#if __WASM__
+            IsStandaloneWindow = WebAssemblyRuntime.InvokeJS(@"window.matchMedia('(display-mode: standalone)').matches") == "true";
+#else
+            IsStandaloneWindow = true;
+#endif
```

Comment:
I wonder if we should be evaluating this on every window size change (when moving from the main window to PWA)

Reply:
@jeromelaban It didn't appear significant towards the user experience to do things that way. I also noticed invoking JS that many times seemed inefficient. In my view, we can iterate on that part later but it seems like a good idea to move forward with these changes now.